### ISSUE TITLE
File open error with empty working dir on ramdrive

### ISSUE THREAD

**Issue by ogmios-voice** [mifi/lossless-cut#2105]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest) (also it might have already been fixed in latest [nightly build](https://github.com/mifi/lossless-cut#nightly-builds-))
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md)
- [X] Search for your problem under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)
- [X] If you have a problem only with a particular file (or file type), [use this form instead](https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&template=file-issue.yml)
- [X] Windows 7, 8 and 8.1 are [no longer supported](https://github.com/mifi/lossless-cut/discussions/1476). **This is not a question about Windows 8.1 or older.**

### Operating System

Windows 11

### Steps to reproduce

1. Create **ramdrive** w/ https://github.com/LTRData/ImDisk
    * `imdisk -a -m r: -t vm -s 4G -p "/fs:ntfs /q /y"` (as admin)
2. Set working dir to ramdrive
4. Ensure there are **no files and no folders** (including hidden ones) on the new drive
5. Open any file (e.g. drop on LosslessCut)

### Expected behavior

File is opened

### Actual behavior

`Error: ENOENT: no such file or directory, scandir 'R:\'`

Update: when at least one file/folder is present on the drive, `scandir` will work properly.

### Share log from developer tools

```js
App.tsx:2079 userOpenFiles Error: ENOENT: no such file or directory, scandir 'R:\'
    at async readdir (node:internal/fs/promises:948:18)
    at async findExistingHtml5FriendlyFile (util.ts:248:22)
    at async App.tsx:1609:41
    at async App.tsx:1722:5
    at async App.tsx:2074:9
    at async HTMLBodyElement.onDrop (App.tsx:2558:7)
```

**Comment by mifi** [comment]
did you try other ram disk solutions? same problem?

I see that:

> PLEASE NOTE: This project is not recommended on recent versions of Windows and many applications written for Windows Vista and later require features that are not supported. It is based on an old design for compatibility with as old versions as Windows NT 3.51. No new features will be added to this project but it will remain available here because it could still be useful in certain scenarios.

Windows 11 is a recent version of Windows, so it could be related to that.

What if you open the command line and run
```bash
r:
dir
```
do you see any files?

**Comment by ogmios-voice** [comment]
I'm using this ramdrive for years now w/o any issue. And it's probably the one of the best ones so it is actively tested by a lot of users.

I did however some more tests, and I've figured out that the problem is w/ `scandir`: when there's absolutely **no file and no folder** on the ramdrive, `scandir` will **fail**. As soon as I create a dummy file on the drive, it will work again.\
I'll update the issue description accordingly.
